backend/kesaseteli/common/utils.py

```diff
@@ -16,21 +16,38 @@ def has_permission(self, request, view):
         return False
 
 
+def has_whitespace(value):
+    value_without_whitespace = "".join(value.split())
+    return value != value_without_whitespace
+
+
+def is_uppercase(value):
+    return value == value.upper()
+
+
 def validate_finnish_social_security_number(value):
     """
-    Raise a ValidationError if the given value is not a Finnish social security number.
+    Raise a ValidationError if the given value is not an uppercase Finnish social
+    security number with no whitespace.
     """
-    if not is_valid_finnish_social_security_number(value):
+    if (
+        not is_valid_finnish_social_security_number(value)
+        or has_whitespace(value)
```

Comment:
`or " " in value` should do the trick here

Reply:
That's a bit different. I want to make sure there is no whitespace at all, not just space characters.

Reply:
ah okay. Got it 👍